libs/game/docs/reference/scene/place-on-random-tile.md

```diff
@@ -6,60 +6,114 @@ Move a sprite's position to the center of a random tile in the scene.
 scene.placeOnRandomTile(null, 0)
 ```
 
-You a can make a sprite to locate itself right on top of a random tile in the tile map. Use the color number of type of tile to select for the sprite to locate on one of them.
-
+You can make a sprite locate itself right on top of a random tile in the tile map. Use the color number of type of tile to select for the sprite to locate on one of them.
```

Comment:
(for the next change) i guess we should eventually take out all the color number stuff since that doesn't apply to the current tilemap blocks?

Reply:
Actually, I wanted to change that paragraph too. I forgot to do that yesterday. Thanks.